cypress-tests/cypress/e2e/PaymentUtils/Xendit.js

```diff
@@ -0,0 +1,624 @@
+const successfulNo3DSCardDetails = {
+  card_number: "4000000000001091",
+  card_exp_month: "12",
+  card_exp_year: "27",
+  card_holder_name: "joseph Doe",
+  card_cvc: "123",
+};
+const billing_details = {
+  email: "mauro.morandi@nexi.it",
+  phone: {
+    number: "9123456789",
+    country_code: "+91",
+  },
+};
+const customer_acceptance = {
+  acceptance_type: "offline",
+  accepted_at: "1963-05-03T04:07:52.723Z",
+  online: {
+    ip_address: "127.0.0.1",
+    user_agent: "amet irure esse",
+  },
+};
+const payment_method_data_3ds = {
+  card: {
+    last4: "1091",
+    card_type: "CREDIT",
+    card_network: "Visa",
+    card_issuer: "INTL HDQTRS-CENTER OWNED",
+    card_issuing_country: "UNITEDSTATES",
+    card_isin: "400000",
+    card_extended_bin: null,
+    card_exp_month: "12",
+    card_exp_year: "27",
+    card_holder_name: "joseph Doe",
+    payment_checks: null,
+    authentication_data: null,
+  },
+  billing: {
+    address: null,
+    email: "mauro.morandi@nexi.it",
+    phone: {
+      number: "8056594427",
```

Comment:
use a generic number
```suggestion
      number: "9123456789",
```